lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -109,7 +109,7 @@ public function bindParam($column, &$variable, $type = ParameterType::STRING, $l
         assert(is_int($column));
 
         if (! isset(self::$_paramTypeMap[$type])) {
-            throw new MysqliException(sprintf("Unknown type: '%s'", $type));
+            throw new MysqliException(sprintf('Unknown type, "%s" given.', $type));
```

Comment:
Unlike the wrapper statement, wrapper statements only accept `ParameterType::*` constants for `$type` which are integers. Even if the type of `$type` is not enforced to `int`, we shouldn't expect anything else here. Therefore, `"%s"` → `%d`.